src/silx/gui/colors.py

```diff
@@ -24,13 +24,21 @@
 """This module provides API to manage colors.
 """
 
+from __future__ import annotations
```

Comment:
This enable using `tuple`, `dict` or `collections.abc.Sequence` as types with Python 3.7.
See [PEP585](https://peps.python.org/pep-0585/#implementation)